test/Core.ts

```diff
@@ -343,6 +343,88 @@ export class Foo {
         })
       })
 
+      it('should replace imports from the project with the local source folder (single quotes)', async () => {
```

Comment:
The two tests differ only in the quote characters. I was thinking of creating a helper function to reduce code duplication, but in the end decided to go for the more literal way as it seemed less confusing. Not sure though.

Reply:
I don't mind the duplication in tests. If we start repeating more then we could abstract.

Reply:
Agreed - I think unless the vast majority of test cases make use of a particular piece of logic, I would actually encourage code duplication to avoid confusion and decrease complexity.